src/main/java/Ellio.java

```diff
@@ -0,0 +1,137 @@
+import java.util.Scanner;
+
+public class Ellio {
+
+    public static final int DEADLINE_INPUT_SPACING = 9;
+    public static final int EVENT_INPUT_SPACING = 6;
+    public static final int MAX_TASK = 100;
+    private static Task[] listTasks = new Task[MAX_TASK];
+    private static int numberTask = 0;
+
+    public static void endProgram(){
+        System.out.println(BotText.lineBorder + BotText.messageGoodbye + BotText.lineBorder);
+    }
+
+    public static void startProgram(){
+        System.out.println(BotText.lineBorder + BotText.messageWelcome + BotText.lineBorder);
+    }
+
+    public static void printList(){
+        System.out.print(BotText.lineBorder + BotText.messageList);
+        for(int i = 0; i < numberTask; i++){
+            System.out.println((i+1) + "." + listTasks[i].getTask());
+        }
+        System.out.println(BotText.lineBorder);
+    }
+
+    public static void addToDo(String line){
+        String description = line.replace("todo ", "");
+        Todo newTodo = new Todo(description);
+        listTasks[numberTask] = newTodo;
+        numberTask++;
+        System.out.println(BotText.lineBorder + "Got it. I've added this task:\n  " + newTodo.getTask());
+        System.out.println("Now you have " + numberTask + " tasks in the list.\n" + BotText.lineBorder);
```

Comment:
Can consider creating a printing class and use a toString method for your individual class, and print using your printing class methods